examples/stats-widget/.eslintrc.js

```diff
@@ -0,0 +1,29 @@
+// prettier-ignore
+module.exports = {
```

Comment:
Add a comment explaining why we need a special eslintrc (presumably because this is the only react using folder)?